src/execution_plan/ops/op_delete.c

```diff
@@ -139,38 +138,36 @@ OpBase *NewDeleteOp(const ExecutionPlan *plan, AR_ExpNode **exps) {
 	op->exp_count = array_len(exps);
```

Comment:
For safety I'll recommend switching from `rm_malloc` to `rm_calloc` above

Reply:
just for one null?

Reply:
Yeah I think it would be better (safety wise, if we adopt this approach)